test/Rules/MOVE_PLAYER.test.js

```diff
@@ -0,0 +1,88 @@
+const expect = require("chai").expect;
+const EventEmitter = require("events");
+const sinon = require("sinon");
+const mockUIFactory = require("../mocks/UI");
+
+const { GameState } = require("../../entities/GameState");
+const { createPlayer } = require("../testutils");
+const config = require("../../config/monopolyConfiguration");
+
+describe("Rules -> MOVE_PLAYER", () => {
+  let gameState;
+  let userInterface;
+  let eventBus;
+  const RULES = require("../../entities/Rules");
+
+  beforeEach(() => {
+    gameState = new GameState();
+    eventBus = new EventEmitter();
+    userInterface = mockUIFactory();
+    gameState.players = [createPlayer({ name: "player1" })];
+    gameState.config = config;
+  });
+
+  afterEach(() => {
+    // Restore the default sandbox here
+    sinon.restore();
+  });
+
+  describe("movePlayer", () => {
+    const inputEvent = "MOVE_PLAYER";
+    const passGoEvent = "PASS_GO";
+
+    let passGoSpy;
+
+    beforeEach(() => {
+      let { emit: notify } = eventBus;
+      notify = notify.bind(eventBus);
+
+      RULES[inputEvent].forEach((handler) =>
+        eventBus.on(
+          inputEvent,
+          handler.bind(null, { notify, UI: userInterface }, gameState)
+        )
+      );
+      gameState.turnValues = {
+        roll: [1, 2],
+      };
+
+      passGoSpy = sinon.spy();
+      eventBus.on(passGoEvent, passGoSpy);
+    });
+
+    it("should make a call to the UI#playerMovement", () => {
+      gameState.currentPlayer.position = 10;
+      const uiSpy = sinon.spy();
+      userInterface.playerMovement = uiSpy;
+      eventBus.emit(inputEvent);
+      expect(uiSpy.calledOnceWithExactly(13)).to.equal(
```

Comment:
I would change this to a `const` from the roll values and the old player position as well

Reply:
Naw, nevermind